ale_linters/dockerfile/dockerfile_lint.vim

```diff
@@ -0,0 +1,66 @@
+" Author: Alexander Olofsson <alexander.olofsson@liu.se>
+
+call ale#Set('dockerfile_dockerfile_lint_executable', 'dockerfile_lint')
+call ale#Set('dockerfile_dockerfile_lint_options', '')
+
+function! ale_linters#dockerfile#dockerfile_lint#GetType(type) abort
+    if a:type is? 'error'
+        return 'E'
+    elseif a:type is? 'warn'
+        return 'W'
+    endif
+
+    return 'I'
+endfunction
+
+function! ale_linters#dockerfile#dockerfile_lint#Handle(buffer, lines) abort
+    try
+        let l:data = json_decode(join(a:lines, ''))
+    catch
+        return []
+    endtry
+
+    if empty(l:data)
+        " Should never happen, but it's better to be on the safe side
+        return []
+    endif
+
+    let l:messages = []
+
+    for l:type in ['error', 'warn', 'info']
+        for l:object in l:data[l:type]['data']
+            try
+                let l:line = l:object['line']
+            catch
+                let l:line = -1
+            endtry
+
+            let l:message = l:object['message']
+
+            if has_key(l:object, 'description') && l:object['description'] != 'None'
```

Comment:
Similar to above: `if get(l:object, 'description', 'None') != 'None'` compacts it to one call.